CONTRIBUTING.md

```diff
@@ -0,0 +1,269 @@
+
+#### Table Of Contents
+
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [ownCloud is hosted on Github](#owncloud-is-hosted-on-github)
+  * [ownCloud Company, Engineering Partners and Community](#owncloud-company,-engineering-partners-and-community)
+  * [Licensing and CLA](licensing-and-cla)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Help spreading the word!](help-spreading-the-word!)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Documentation Contributions](#documentation-contributions)
+  * [Internationalization](#internationalization)
+  * [Deployments: Docker, Ansible and friends](#deployments:-docker,-ansible-and-friends)
+
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Golang Styleguide](#golang-styleguide)
+  * [Web Styleguide](#web-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+[Additional Notes](#additional-notes)
+  * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+  ## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** Please don't file an issue to ask a question. You'll get faster results by using the resources below.
+
+For general questions, please refer to [ownCloud's FAQs](https://owncloud.com/faq/) or ask on the [ownCloud Central Server](https://central.owncloud.org/).
+
+We also have a [Rocket Chat Server](https://talk.owncloud.com/channel/infinitescale) to answer your questions specifically to ownCloud Infinite Scale.
+
+## What should I know before I get started
+
+### ownCloud is hosted on Github.
+
+To effectivly contribute to ownCloud Infinite Scale, you should make sure you have an Github account. You can get that for free at [Github](https://github.com/join). You can find howtos on the internet, for example [here](https://www.wikihow.com/Create-an-Account-on-GitHub).
+
+For other ways of contributing, for example with translations, other systems require you to have an account, for example [Transifex](https://www.transifex.com)
+
+The ownCloud development follows the strict Github based workflow of
+
+### ownCloud Company, Engineering Partners and Community
+
+ownCloud Infinite Scale is heavily developed developers who are employed by the [ownCloud company](https://www.owncloud.com), which is located in Germany. It is providing support for ownCloud for customers worldwide. In addition there are engineering partners who also work full time on ownCloud related code, for example in [CERN REVA](https://github.com/cs3org/reva/).
```

Comment:
```suggestion
ownCloud Infinite Scale is largely created by developers who are employed by the [ownCloud company](https://www.owncloud.com), which is located in Germany. It is providing support for ownCloud for customers worldwide. In addition there are engineering partners who also work full time on ownCloud related code, for example in [CERN REVA](https://github.com/cs3org/reva/).
```

Reply:
+ `providing support for ownCloud software _to_ customers worldwide`